build-support/bin/generate_travis_yml.py

```diff
@@ -722,7 +724,7 @@ def _osx_sanity_check(
         **osx_shard(python_version=python_version, osx_image=osx_image),
         "name": f"OSX 10.{os_version_number} sanity check (Python {python_version.decimal})",
         "script": [
-            f"MODE=debug ./build-support/bin/ci.py --sanity-checks --python-version {python_version.decimal}"
```

Comment:
Does this shard not use the `GLOBAL_ENV_VARS`? If it uses them, it should probably continue to override back to debug here.

Reply:
I'm not sure why we were configuring it to begin with. You must explicitly call `ci.py --bootstrap` to bootstrap Pants. Otherwise, it will use `pants.pex` and fail if it can't find that.

Reply:
Ah, k.